package.json

```diff
@@ -8,23 +8,24 @@
     "buoy": "./bin/buoy.js"
   },
   "scripts": {
-    "watch": "cross-env TARGET=development webpack --config build/config.js --progress --hide-modules --watch",
-    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.dev.config.js --open --hot",
     "build": "npm run build:dist && npm run build:es5",
     "build:dev": "cross-env NODE_ENV=production node build/webpack.dev.config.js",
     "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config build/config.js --progress --hide-modules",
     "build:es5": "rimraf es5 && cross-env NODE_ENV=es5 babel src --out-dir es5",
     "debug-build": "node --inspect --debug-brk build/config.js",
     "debug:test": "./node_modules/.bin/cross-env NODE_ENV=test node --inspect --inspect-brk ./node_modules/jest/bin/jest.js --no-cache --runInBand --verbose",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.dev.config.js --open --hot",
+    "lint": "eslint --ext .js,.vue src",
+    "lint:fix": "eslint --fix --ext .js,.vue src",
     "test": "cross-env NODE_ENV=test jest",
```

Comment:
I saw a comment about this with `-i` but never saw an answer.

Reply:
As in? Not running them serially?

Reply:
It sounds like `-i` was removed in a previous PR and a question was asked about jest running tests serially or not. Is this something we need to talk about or is it not a concern and we can move on?

Reply:
I suspect components don't interact with each other across tests (since these are unit tests). From Jest documentation:
```
./node_modules/.bin/jest --help | grep -A 10 "\-i"
  --runInBand, -i                 Run all tests serially in the current process
                                  (rather than creating a worker pool of child
                                  processes that run tests). This is sometimes
                                  useful for debugging, but such use cases are
```

It was there and was not needed so I deleted it

Reply:
Sounds good to me